companion/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+	"id": "disguise",
+	"name": "disguise",
+	"shortname": "d3-osc",
+	"description": "OSC plugin for Companion",
+	"version": "3.0.0",
```

Comment:
The last release was 1.1.0, so any major version bump would take it to 2.0.0, not 3.0.0